packages/extension/src/contexts/LedgerContext/LedgerContext.tsx

```diff
@@ -75,15 +87,15 @@ const LedgerProvider: FC = ({ children }) => {
         // Prepare the transaction
         const prepared = await client.request({
           command: 'account_nfts',
-          account: wallet.publicAddress,
+          account: 'rDGh681kc6V1GKkQB378XhiM1tzkYrnFwQ', // Change before merging to master
```

Comment:
I think you forgot to revert this change :)

Reply:
Just keeping here for testing until everything else is ready, will remove it after that :)